Header header
Blob[] blobs
uint32 image_height
uint32 image_width